src/components/Combobox/Combobox.tsx

```diff
@@ -243,6 +250,28 @@ function Combobox<Item>({
               isOpen={isOpen && results.length > 0}
               {...getMenuProps()}
             >
+              {withClearSelectionControl && (
```

Comment:
I am not sure we should place this option as a list item, I would suggest moving the clear option outside of the list, thus we could avoid the `sticky` positioning and scroll through the actual results.

Reply:
Can you provide an example implementation? Right now the `Menu` is the one animating and containing all options. If I don't put this insidee the `Menu` component it won't animate or show correctly.